cookbook/configuration/pdo_session_storage.rst

```diff
@@ -120,10 +119,16 @@ configuration format of your choice):
         ));
         $container->setDefinition('session.handler.pdo', $storageDefinition);
 
-* ``db_table``: The name of the session table in your database
-* ``db_id_col``: The name of the id column in your session table (VARCHAR(255) or larger)
-* ``db_data_col``: The name of the value column in your session table (TEXT or CLOB)
-* ``db_time_col``: The name of the time column in your session table (INTEGER)
+These are parameters that you must configure:
+
+``db_table``
+    The name of the session table in your database.
+``db_id_col``
+    The name of the id column in your session table (``VARCHAR(255)`` or larger).
+``db_data_col``
+    The name of the value column in your session table (``TEXT`` or ``CLOB``).
+``db_time_col``:
+    The name of the time column in your session table (``INTEGER``).
```

Comment:
@weaverryan IIRC, we need to make sure that this is updated properly when merged into the `2.6` branch (there is one additional column since Symfony 2.6).